opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -149,7 +221,7 @@ def __init__(
         trace_config: None = None,  # TODO
         resource: None = None,
         attributes: types.Attributes = None,  # TODO
-        events: Sequence[trace_api.Event] = None,  # TODO
+        events: Sequence[Event] = None,  # TODO
```

Comment:
Shouldn't `Tracer.start_span` and `Tracer.start_as_current_span` take in `events` as well in the api? I think if this is the case, you have to move the `Event` definition back from the sdk to the api.

Reply:
According to https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#span-creation the events cannot be passed at start creation time, infact we should remove them from the constructor.

Reply:
Makes sense. Curious, what does the `# TODO` refer to in the code? Can that be removed with these changes?

Reply:
I don't know what the TODO was about. I'll remove that in a followup PR in next days.